src/Microsoft.Data/DataFrame.cs

```diff
@@ -500,27 +511,33 @@ public DataFrame DropNulls(DropNullOptions options = DropNullOptions.Any)
             return this[filter];
         }
 
-        public DataFrame FillNulls(object value)
+        public DataFrame FillNulls(object value, bool inPlace = false)
         {
-            DataFrame ret = new DataFrame();
-            for (int i = 0; i < ColumnCount; i++)
+            DataFrame ret = this;
+            if (!inPlace)
+                ret = Clone();
+
+            for (int i = 0; i < ret.ColumnCount; i++)
             {
-                ret.InsertColumn(i, Column(i).FillNulls(value));
+                ret.Column(i).FillNulls(value, true);
```

Comment:
Similar comment here about naming boolean parameters for easy reading.  `inPlace: true`.